src/quo2/foundations/colors.cljs

```diff
@@ -117,6 +117,9 @@
 
 ;;;;Black
 
+;;OnBoarding
+(def onboarding-black "#000716")
```

Comment:
I want to check with design team what this color is and if there's a more generic name we can use 👍